mlir/utils/jenkins/Jenkinsfile

```diff
@@ -389,6 +407,8 @@ pipeline {
                      description: 'Run the MIGraphX integration tests')
         booleanParam(name: 'checkCK', defaultValue: true,
                      description: 'Compare performance against CK')
+        booleanParam(name: 'runCodeCoverage', defaultValue: params.weekly || (!params.weekly && !params.nightly),
```

Comment:
New parameter.  Defaults to run on weeklies and pull requests.  The logic at codecov.io does tracking of changes that should be good for pull requests.